packages/apputils-extension/src/announcements.ts

```diff
@@ -153,6 +153,14 @@ export const announcements: JupyterFrontEndPlugin<void> = {
             ]
           }
         );
+        let scanModeOffRegion = document.getElementById(
+          'screen-reader-aria-live'
+        );
+        scanModeOffRegion!.append(
```

Comment:
Why is it only shown when `mustFetchNews` is set to none? Is it the right place to add this message?

Reply:
We have placed it with notifications but moved it out of the `mustFetchNews === 'none' `